### PR TITLE
fix(e2e-tests): tests for staking-info endpoint

### PR DESCRIPTION
**Pull request by TarikGul**
rel: [#741](https://github.com/paritytech/substrate-api-sidecar/pull/741)

Follow up to the above PR.
Adds the e2e tests for staking-info

### CODE REVIEW

File: e2e-tests/endpoints/kusama/accounts/staking-info/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		JSON.stringify(accountsStakingInfo5000000),
+	], // v2026
+	[
+		'/accounts/HP8qJ8P4u4W2QgsJ8jzVuSsjfFTT6orQomFD6eTRSGEbiTK/staking-info?at=8000000',
+		JSON.stringify(accountsStakingInfo8000000),
+	], // v9040
+	[
+		'/accounts/HP8qJ8P4u4W2QgsJ8jzVuSsjfFTT6orQomFD6eTRSGEbiTK/staking-info?at=9500000',
+		JSON.stringify(accountsStakingInfo9500000),
+	], // v9090
```
Comment (niklasad1) [inline e2e-tests/endpoints/kusama/accounts/staking-info/index.ts:28]: v9120 was never deloyed on kusama?!

Comment (TarikGul) [reply e2e-tests/endpoints/kusama/accounts/staking-info/index.ts:28]: Yea 9120 was never deployed to kusama, it went straight from 9111, and then to 9122. A good place to reference for these is inside of the [`types-known`](https://github.com/polkadot-js/api/blob/master/packages/types-known/src/upgrades/kusama.ts) package. Handy documentation of all the upgrades for each chain.

**Comment by TarikGul** [comment]
@jsdw, yea I can do a second double check. :)

**Review by jsdw — APPROVED** [review]

**Comment by jsdw** [comment]
I've hit approve, but yeah, I'm not so sure what to look out for myself on this one :)

**Review by insipx — APPROVED** [review]

**Comment by TarikGul** [comment]
@jsdw Yea the e2e test PR's wont have much to Review in terms of logic. Its really just hardcoding json responses and injecting them into the e2e tests. The reason I separated them now from the main PR is because I find when I add the e2e tests in the same PR as the `update historicApi` it tends to be cluttered, and probably harder to review for everyone on the team.

**Comment by TarikGul** [comment]
Cool just double checked and all tests passed, merging.

**Pull request merged**